CHANGELOG.md

```diff
@@ -20,14 +20,34 @@ This release has the following breaking changes:
   - The ZIP-317 convention fee increases based on the number of logical transparent or
     shielded actions in a transaction.
   - This change has no impact under normal network conditions.
-- TODO: search the changelog for breaking changes
+- Use HTTP to clone Zebra in Docker documentation (https://github.com/ZcashFoundation/zebra/pull/5529)
```

Comment:
I'm not sure if changing the docs is a breaking change? Which use case does it break?

Reply:
I think I confused this one with the changed Docker command, but we can remove

Reply:
```suggestion
```